src/views/AccessControl.tsx

```diff
@@ -237,9 +243,11 @@ export const AccessControl = () => {
         dispatch(ruleActions.setRule({
             ID: ruleToAction?.ID || null,
             Name: ruleToAction?.Name,
+            Description: ruleToAction?.Description,
             Source: ruleToAction?.Source,
             Destination: ruleToAction?.Destination,
-            Flow: ruleToAction?.Flow
+            Flow: ruleToAction?.Flow,
```

Comment:
Codacy has a fix for the issue: [Delete `····`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
        Flow: ruleToAction?.Flow,
```